modules/tinymce/src/core/main/ts/paste/CutCopy.ts

```diff
@@ -21,22 +21,14 @@ interface SelectionContentData {
 type DoneFn = () => void;
 type FallbackFn = (html: string, done: DoneFn) => void;
 
-const hasWorkingClipboardApi = (clipboardData: DataTransfer | null): clipboardData is DataTransfer =>
-  // iOS supports the clipboardData API but it doesn't do anything for cut operations
-  !Env.os.isiOS() && typeof clipboardData?.setData === 'function';
```

Comment:
I tested this and this is no longer applicable as iOS does work with cut/copy/paste, as such I removed it.